server/src/ClientToDBAPI/IntersectionsRoute/mainRoute.ts

```diff
@@ -45,7 +45,7 @@ export const handleDBErrors = (response: Response, errorMsg: string) => {
             user: response.locals.user.id
         });
     }
-    response.status(errorStatusCode).send(errorMsg);
+    response.send(errorMsg);
```

Comment:
Why sending error message in 200 code?

Reply:
this is what fixes the bug that caused in mabar, because of DP I can't send a response with 500 if everything works, so even though I'm sending 500, DP takes the response and sends 200. So either way, the client gets 200